.github/workflows/test.yml

```diff
@@ -46,6 +47,10 @@ jobs:
             gemfile: gemfiles/activerecord_6.1.gemfile
           - ruby: "3.3"
             gemfile: gemfiles/activerecord_7.0.gemfile
+          - ruby: "2.7"
```

Comment:
Rails 7.2 supports Ruby 3.1+.
https://edgeguides.rubyonrails.org/7_2_release_notes.html#make-ruby-3-1-the-new-minimum-version